meta-lmp-support/files/pelion-passwd-table

```diff
@@ -0,0 +1,7 @@
+parsec:x:1004:1004::/:/sbin/nologin
+maestro:x:1200:900::/:/sbin/nologin
+deviceos:x:1201:901::/:/sbin/nologin
+developer:x:1202:1022::/:/sbin/nologin
+support:x:1203:1023::/:/sbin/nologin
+user:x:1204:1024::/:/sbin/nologin
+developer2:x:1212:1020::/:/sbin/nologin
```

Comment:
Is nologin corrrect for all of these users? Seems better in terms of security, but I do not know if anything relies on them.

Reply:
@petedyerarm does this mean that only `fio` user is allowed to login? and I think by default root login is disabled.

Reply:
That is my understanding. 
Travis indicated that some of these users should be removed, but that could be done later.